### ISSUE TITLE
Multi-Line grammar support for import definitions

### ISSUE THREAD

**Issue by abhishek-kumar0409** [tliron/puccini#16]
imports:
     - datatypes: 
            file: my_custom_vnfd_datatypes.yaml
     - Custom Definitions:
            file: my_custom_vnfd_extensions.yml
     - Custom node Types:
             file: my_custom_node_types_extensions.yaml

Problems(6):

     imports[0].**datatypes**: field is **unsupported**
     imports[0].file: field is required
     imports[1].**Custom Definitions**: field is **unsupported**
     imports[1].file: field is required      
     imports[2].**Node extensions**: field is **unsupported**
     imports[2].file: field is required      

Due to IP rights constraints, I can't share the sample VNFDs but I can point you to grammar as  per the Tosca specifications.
**Grammar reference:**  [Section-3.5.8.2 of TOSCA -v1.2 spec](http://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.2/csd01/TOSCA-Simple-Profile-YAML-v1.2-csd01.html)

**Comment by tliron** [comment]
OK, this seems like yet another case of the spec being inconsistent.

Section 3.5.8.2, which you quote, actually has this structure:
```
imports:
 - file: my_custom_vnfd_datatypes.yaml
 - file: my_custom_vnfd_extensions.yml
 - file: my_custom_node_types_extensions.yaml
```
And that indeed is what Puccini supports.

However, section 3.5.8.3 has an example where it uses the syntax you suggest above (a sequenced list).

I think that 3.5.8.3 is an error and should be ignored because 1) I can't find examples like this in other parts of the spec, and 2) it's unnecessary to give imports a name, because in fact the name is not used anywhere in the grammar.

**Comment by abhishek-kumar0409** [comment]
Yes, Tal... Sorry my reference was wrong(it should be 3.5.8.3).But, I do agree with you because of the ambiguity. We have couple of VNF descriptors from different sources and all of them follow the syntax referred in section 3.5.8.3. It seems to be controversial because there are two ways to represent the multi-line grammar and I'm not sure which one is right or wrong. As a common practice, the readers will try to refer the examples. 
I understand it's unnecessary to give the import names but it's just for readability purpose to describe about the imported file. There are lot of tosca_definitions which are not approved yet or present in SOL_001 spec.  That's why, the VNF providers are providing their custom extensions in this fashion. 

Do we need to reach out to someone for more clarifications. What do you suggest ?

**Comment by abhishek-kumar0409** [comment]
Tal,
Please close this issue as well. There was an inconsistency between the examples and the schema definitions in TOSCA spec. It's better not to provide any quirk support for this thing.

Thanks much !!
